spec/support/comprehensive_mocking.rb

```
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
```

WebMock.disable_net_connect ensures tests fail fast if any external service isn't mocked